tox.ini

```diff
@@ -15,12 +15,13 @@ deps =
    coverage
    isort
    mccabe
+   pytest
 
 setenv =
     COVERAGE_FILE = {toxinidir}/.coverage.{envname}
 
 commands =
-    python -Wi {envsitepackagesdir}/coverage run -m unittest discover -s {envsitepackagesdir}/pylint/test/ -p {posargs:*test_*}.py
+    python -Wi {envsitepackagesdir}/coverage run -m pytest {envsitepackagesdir}/pylint/test/
```

Comment:
I guess I don't understand why we would need this.  What filtering would you want to do at this level?